src/pages/chat/Chat.tsx

```diff
@@ -85,22 +99,22 @@ const Chat = () => {
   }
 
   const selectChatChip = (
-    <AppChip labelSx={styles.chipLabel} sx={styles.chip}>
+    <AppChip labelSx={styles.chipLabel(false)} sx={styles.chip}>
       {t('chatPage.chat.chipLabel')}
     </AppChip>
   )
 
-  const scrollableContent = messagesLoad ? (
-    <Loader pageLoad size={50} sx={styles.loader} />
-  ) : (
-    <SimpleBar
-      scrollableNodeProps={{ ref: scrollRef }}
-      style={styles.scrollableContent}
-    >
-      <ChatDate date={new Date()} />
-      {messagesList}
-    </SimpleBar>
-  )
+  const scrollableContent =
+    messages.length > 0 ? (
```

Comment:
it can be changed just to `messages.length`

Reply:
you are right, fixed